uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -113,13 +114,12 @@ private void addDataToFile(String data, Path filePath) throws IOException {
             writer.write(data + "\r\n");
         }
     }
-
     /**
```

Comment:
Add this space back checkstyle will complain. Make sure you compile before pushing